pkg/daemon/rpm-ostree.go

```diff
@@ -260,10 +261,11 @@ func (r *RpmOstreeClient) Rebase(imgURL, osImageContentDir string) (changed bool
 func runGetOut(command string, args ...string) ([]byte, error) {
 	glog.Infof("Running captured: %s %s", command, strings.Join(args, " "))
 	cmd := exec.Command(command, args...)
-	cmd.Stderr = os.Stderr
+	var stderr bytes.Buffer
+	cmd.Stderr = &stderr
 	rawOut, err := cmd.Output()
```

Comment:
I think it'd be both simpler and better to use https://golang.org/pkg/os/exec/#Cmd.CombinedOutput in this case - this way stdout/stderr will be ordered correctly, and we only have one buffer to look at.

Reply:
agree. Fixed it in latest push.